### DISCUSSION TITLE
UX not finding .net

### DISCUSSION THREAD

**Discussion by cyb3rv0id** [OpenTabletDriver/OpenTabletDriver#1866] [Q&A] [answered]
So I have the .net x64 all downloaded and installed but everytime I try to open up the OpenTabletDriver.UX.Wpf file it gives me a pop up saying this

<img width="334" alt="popup" src="https://user-images.githubusercontent.com/97084344/148022138-3e026260-301b-41e2-9ca7-c4370f348390.png">

 and I dont know why nothing is working

**Comment by Mrcubix** [comment, marked as answer]
# The Correct .NET Version
Windows (x64 Installer): [Windows: Desktop Runtime (v5.0.12)](https://download.visualstudio.microsoft.com/download/pr/1daf85dc-291b-4bb8-812e-a0df5cdb6701/85455a4a851347de26e2901e043b81e1/windowsdesktop-runtime-5.0.12-win-x64.exe)
MacOS (x64 Installer): [MacOS: Runtime (v5.0.12)](https://download.visualstudio.microsoft.com/download/pr/0439a1bb-d460-481c-8417-421e1d9d35c7/871e3f7dcdb93a40c65b1aeb759ff916/dotnet-runtime-5.0.12-osx-x64.pkg)
Linux (x64 Binaries): [Linux: Runtime (v5.0.12)](https://download.visualstudio.microsoft.com/download/pr/781b7ae6-166c-4114-97f8-926d2bf74d34/fe51479e3138d672c512ef0322be23d3/dotnet-runtime-5.0.12-linux-x64.tar.gz)
Linux Package manager: [Install .NET on Linux](https://docs.microsoft.com/en-us/dotnet/core/install/linux) (Do not use Snap package)
